test/integration/test_win_group1.yml

```diff
@@ -9,3 +9,4 @@
     - { role: win_fetch, tags: test_win_fetch }
     - { role: win_regmerge, tags: test_win_regmerge }
     - { role: win_regedit, tags: test_win_regedit }
+    - { role: win_owner, tags: test_win_owner }
```

Comment:
The `test_win_group` prefixed files are used for the legacy method of running Windows tests and don't need to be updated. They probably won't be around much longer.